frontend/Draggable.js

```diff
@@ -15,8 +15,8 @@ var ReactDOM = require('react-dom');
 import type {DOMEvent} from './types';
 
 class Draggable extends React.Component {
-  _onMove: (evt: DOMEvent) => void;
-  _onUp: (evt: DOMEvent) => void;
+  _onMove: (evt: any) => void;
+  _onUp: (evt: any) => void;
```

Comment:
I tried that, and it doesn't seem to do the trick. 

See the error here: https://gist.github.com/tomaszlakomygp/71153238662af3781732a0ff82597058